src/modules/sidebar/sections/SidebarSharing.vue

```diff
@@ -38,6 +41,13 @@ export default {
 
 	computed: {
 		...mapState(useTablesStore, ['activeElement', 'isView']),
+		currentUrl() {
+			const url = this.activeElement
```

Comment:
Something Nextcloud specific, we should use [generateUrl](https://nextcloud-libraries.github.io/nextcloud-router/functions/generateUrl.html) here to make sure this also works properly if Nextcloud was installed in a subdirectory.